packages/core/src/stories/announcements/table-libraries.stories.ts

```diff
@@ -0,0 +1,116 @@
+import type { Meta } from '@storybook/html';
+
+const meta: Meta = {
+  title: 'Intro/Announcements',
+  parameters: {
+    layout: 'fullscreen',
+    options: {
+      showPanel: false,
+      showToolbar: false,
+    },
+  },
+};
+
+export default meta;
+export const TableLibraries = {
+  render: () => `<style>
+            article {
+                box-sizing: border-box;
+                max-width: 688px;
+                padding: 32px;
+                margin: auto;
+                > * {
+                    margin-bottom: 72px;
+                }
+            }
+
+            section > p,
+            section > ul,
+            section > tds-link {
+                margin-bottom: 32px;
+            }
+
+            tds-link {
+                display: inline-block;
+            }
+
+            .mb-72 {
+                margin-bottom: 72px;
+            }
+            </style>
+
+            <article class="tds-u-p2 tds-body-01">
+                <section>
+                    <h2 class="tds-headline-02">Tegel Table</h2>
+                    <p> The Tegel Table component is designed to be a flexible and customizable table solution. It does not include built-in logic but instead relies on events and user-defined custom logic. On our demo pages, we have provided examples of how to implement basic logic for the Tegel Table. </p>
+                    <h5>Highlights of Tegel Table:</h5>
+                    <ul>
+                        <li>Part of the Tegel Design System, free to use.</li>
+                        <li>Simple UI components with custom events, props, and slots to accommodate different use cases and tech stacks within Scania.</li>
+                        <li>Sorting, multiselect, filtering, and pagination features with logic are presented on the demo pages.</li>
+                    </ul>
+                    <h5>What do I need to install to use Tegel Table?</h5>
+                    <ul>
+                        <li>Tegel Table is part of the Tegel Design System, so no additional installation is needed.</li>
```

Comment:
I'm thinking, to say "Tegel Table" is redundant, as it's name is not "Tegel Table", merely referring to it as the "Table component" suffice.

Reply:
I agree. Thanks Tim for suggestion, made it so easy just to commit it asap ;)